### ISSUE TITLE
Document Locks

### ISSUE THREAD

**Issue by uilianries** [conan-io/docs#924]
Hi!

I'm not really familiar with **locks** files and I didn't find any information about it on docs.conan. I think would be nice a section explaining about those lock files and what they represent. I saw some question about locks on Slack but I don't have idea how to respond.

Conan Version: 1.8.4

**Comment by danimtb** [comment]
Agree! It should be explained at least here: https://docs.conan.io/en/latest/reference/commands/misc/remove.html
